SCons/Script/SConscript.py

```diff
@@ -202,6 +202,7 @@ def _SConscript(fs, *files, **kw):
                 else:
                     f = fs.File(str(fn))
                 _file_ = None
+                f.is_sconscript = True
```

Comment:
You'd need to add this to __slots__

Reply:
I added it to the __slots__ in SCons.Node.Node class which is the base node class. Was there more to update?